<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>518</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255,255,255);
background-color: rgb(245, 245, 245);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea { border: none; }

/* Style for vertical scrollbar */
QScrollBar:vertical {
    border: none;
    background: #E0E0E0; /* Scrollbar background */
    width: 12px;
    margin: 0px;
    border-radius: 6px; /* Makes the scrollbar round */
}

/* Handle of the vertical scrollbar */
QScrollBar::handle:vertical {
    background-color: #888888; /* Scrollbar handle color */
    min-height: 20px;
    border-radius: 6px; /* Round handle */
}

/* Top and bottom arrows */
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: none;
    background: none;
    height: 0px;
}

/* Top and bottom space before and after handle */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

/* Style for horizontal scrollbar */
QScrollBar:horizontal {
    border: none;
    background: #E0E0E0; /* Scrollbar background */
    height: 12px;
    margin: 0px;
    border-radius: 6px; /* Makes the scrollbar round */
}

/* Handle of the horizontal scrollbar */
QScrollBar::handle:horizontal {
    background-color: #888888; /* Scrollbar handle color */
    min-width: 20px;
    border-radius: 6px; /* Round handle */
}

/* Left and right arrows */
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    border: none;
    background: none;
    width: 0px;
}

/* Left and right space before and after handle */
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}

</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>930</width>
         <height>978</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>960</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <widget class="QTextEdit" name="logTextEdit">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>510</y>
             <width>891</width>
             <height>381</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">    QTextEdit {
        border: 2px solid #4A90E2;      /* Border color (adjust to your design) */
        border-radius: 10px;            /* Adjust the radius for rounded corners */
        padding: 5px;                   /* Padding inside the line edit */
        background-color: #FFFFFF;      /* Background color */
        color: #333333;                 /* Text color */
    }

    QTextEdit:focus {
        border-color: #50C878;          /* Border color when the line edit is focused */
        background-color: #F5F5F5;      /* Background color when focused */
    }</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="ramUsageLabel">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>460</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(51, 51, 51);</string>
           </property>
           <property name="text">
            <string>Ram Usage</string>
           </property>
          </widget>
          <widget class="QPushButton" name="startButton">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>110</y>
             <width>130</width>
             <height>130</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"> QPushButton {
        border: 2px solid #4A90E2;      /* Border color */
        border-radius: 60px;            /* Round corners */
        padding: 8px 16px;              /* Padding for size and spacing */
        background-color: #4A90E2;      /* Button background color */
        color: white;                   /* Text color */
        font-size: 14px;                /* Font size */
    }

    QPushButton:hover {
        background-color: #50C878;      /* Background color on hover */
        border-color: #50C878;          /* Border color on hover */
    }

    QPushButton:pressed {
        background-color: #2ECC71;      /* Background color when pressed */
        border-color: #2ECC71;          /* Border color when pressed */
    }

    QPushButton:disabled {
        background-color: #E0E0E0;      /* Background color when disabled */
        color: #7F8C8D;                 /* Disabled text color */
        border-color: #E0E0E0;          /* Border color when disabled */
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="commandInput">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>910</y>
             <width>791</width>
             <height>24</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">    QLineEdit {
        border: 2px solid #4A90E2;      /* Border color (adjust to your design) */
        border-radius: 10px;            /* Adjust the radius for rounded corners */
        padding: 5px;                   /* Padding inside the line edit */
        background-color: #FFFFFF;      /* Background color */
        color: #333333;                 /* Text color */
    }

    QLineEdit:focus {
        border-color: #50C878;          /* Border color when the line edit is focused */
        background-color: #F5F5F5;      /* Background color when focused */
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="executeButton">
           <property name="geometry">
            <rect>
             <x>820</x>
             <y>910</y>
             <width>80</width>
             <height>24</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"> QPushButton {
        border: 2px solid #4A90E2;      /* Border color */
        border-radius: 10px;            /* Round corners */
        background-color: #4A90E2;      /* Button background color */
        color: white;                   /* Text color */
    }

    QPushButton:hover {
        background-color: #50C878;      /* Background color on hover */
        border-color: #50C878;          /* Border color on hover */
    }

    QPushButton:pressed {
        background-color: #2ECC71;      /* Background color when pressed */
        border-color: #2ECC71;          /* Border color when pressed */
    }

    QPushButton:disabled {
        background-color: #E0E0E0;      /* Background color when disabled */
        color: #7F8C8D;                 /* Disabled text color */
        border-color: #E0E0E0;          /* Border color when disabled */
    }</string>
           </property>
           <property name="text">
            <string>&lt;-</string>
           </property>
          </widget>
          <widget class="NHLabel" name="Logo">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>10</y>
             <width>64</width>
             <height>64</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>GameServe</string>
           </property>
          </widget>
          <widget class="NHLabel" name="GameServe">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>60</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>GAMESERVE</string>
           </property>
          </widget>
          <widget class="QFrame" name="speedometerPlaceholder">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>280</y>
             <width>180</width>
             <height>180</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>800</x>
             <y>30</y>
             <width>80</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NHLabel</class>
   <extends>QLabel</extends>
   <header>nhlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
